src/patternfly/components/FormControl/form-control.hbs

```diff
@@ -1,4 +1,7 @@
-<{{ controlType }} class="pf-c-form-control{{#if pf-c-form-control--modifier}} {{pf-c-form-control--modifier}}{{/if}}"
+<{{ controlType }} class="pf-c-form-control{{#if form-control--modifier}} {{form-control--modifier}}{{/if}}"
+{{#if form-control--attribute}}
+    {{{form-control--attribute}}}
+{{/if}}
 {{#if type}} type="{{type}}"{{/if}}
 {{#if placeholder}} placeholder="{{placeholder}}"{{/if}}
```

Comment:
This is my bad I should have updated this when I did the form updates.
because we are adding form-control--attribute we can remove the conditionals for type,placeholder,value,name.readonly, ariadescribedby,aria-label,aria-errormessage,required,aria-invalid,disabled,id, multiple and update the examples to wrap those attributes in form-control--attribute.